infrastructure.tf

```diff
@@ -25,22 +25,22 @@ variable master_memory {
     description = "The amount of memory available for the shape of each master node, in gigabytes. The default value is 16. "
 }
 variable master_boot_size {
-    default     = 500
+    default     = 1024
     type        = number
-    description = "The size of the boot volume of each master node in GBs. The minimum value is 50 GB and the maximum value is 32,768 GB (32 TB). The default value is 500 GB. "
+    description = "The size of the boot volume of each master node in GBs. The minimum value is 50 GB and the maximum value is 32,768 GB (32 TB). The default value is 1024 GB. "
 }
 variable master_boot_volume_vpus_per_gb {
-    default     = 60
+    default     = 90
     type        = number
-    description = "The number of volume performance units (VPUs) that will be applied to this volume per GB of each master node. The default value is 60. "
+    description = "The number of volume performance units (VPUs) that will be applied to this volume per GB of each master node. The default value is 90. "
 }
 variable worker_count {
-    default     = 3
+    default     = 4
```

Comment:
@davidfoster0127 I believe 4 nodes was targeting the OPCT/validation environment, maybe we don't want to change the default for end-users. WDYT?

Reply:
Correct, changing back to 3.